src/screenControl.py

```diff
@@ -100,6 +105,38 @@ def getIsSidebarMode(self):
         (maxy, maxx) = self.screenControl.getScreenDimensions()
         return maxx > 200
 
+    def outputDescriptionPane(self, lineObj):
+        if not self.getIsSidebarMode():
+            return
+        (maxy, maxx) = self.screenControl.getScreenDimensions()
+        borderX = maxx - self.WIDTH
+        startY = self.SIDEBAR_Y + 1
+        headerLine = 'Description for ' + lineObj.path + ' :'
+        descLines = [
+                        lineObj.getTimeLastAccessed(),
+                        lineObj.getTimeLastModified(),
+                        lineObj.getOwnerId(),
+                        lineObj.getSizeInBytes(),
+                        lineObj.getLengthInLines()
+                    ]
+        self.printer.addstr(startY, borderX + 2, headerLine)
+        y = startY + 2
+        for descLine in descLines:
+            self.printer.addstr(y, borderX + 2, '    * ' + descLine)
+            y = y + 1
+        self.DESCRIPTION_CLEAR = False
+
+    # to fix bug where description pane may not clear on scroll
+    def clearDescriptionPane(self):
+        if self.DESCRIPTION_CLEAR:
+            return
+        (maxy, maxx) = self.screenControl.getScreenDimensions()
+        borderX = maxx - self.WIDTH
+        startY = self.SIDEBAR_Y + 1
+        for i in range(startY, maxy - 1):
+            self.printer.clearseg(i, borderX + 2, maxx)
```

Comment:
maybe it makes sense to do a `clearSquare` method that takes in the corners? since thats how we use it here